atom/browser/notifications/mac/cocoa_notification.mm

```diff
@@ -29,7 +27,9 @@
   notification_.reset([[NSUserNotification alloc] init]);
 
   NSString* identifier =
-      [NSString stringWithFormat:@"ElectronNotification%d", g_identifier_++];
+      [NSString stringWithFormat:@"%@:notification:%@",
```

Comment:
does this colon-separated syntax have any particular meaning?

Reply:
@nornagon nope, just makes it more readable in the debug messages further down in this file